src/servers/ZoneServer2016/managers/decaymanager.ts

```diff
@@ -61,6 +64,37 @@ export class DecayManager {
     }, this.decayTickInterval);
   }
 
+  private constructionGriefCheck(server: ZoneServer2016) {
```

Comment:
Btw why not doing that stuff at the beginning of contructionExpirationCheck method ? to avoid looping two time in a row on the same data?

Reply:
#1859